scripts/jenkins/ardana/ansible/roles/heat_stack/defaults/main.yml

```diff
@@ -15,4 +15,8 @@
 #
 ---
 
-ses_rgw_port: "{{ hostvars['ardana-' + qe_env].ses_rgw_port }}"
+os_cloud: "engcloud-cloud-ci"
+heat_action: "create"
+api_timeout: "3600"
```

Comment:
Could you set this back to 10 minutes, as it was the case with the old bash script this is replacing ?

Reply:
done!